WalletWasabi/Blockchain/Keys/BlockchainState.cs

```diff
@@ -10,19 +10,21 @@ namespace WalletWasabi.Blockchain.Keys;
 public class BlockchainState
 {
 	[JsonConstructor]
-	public BlockchainState(Network network, Height height)
+	public BlockchainState(Network network, Height height, Height turboSyncHeight)
 	{
 		Network = network;
 		Height = height;
+		TurboSyncHeight = turboSyncHeight;
 	}
 
 	public BlockchainState()
 	{
 		Network = Network.Main;
 		Height = 0;
+		TurboSyncHeight = 0;
```

Comment:
Here we do `TurboSyncHeight = 0;` while everywhere else we use `Height.Unknown`.